.travis.yml

```diff
@@ -5,9 +5,21 @@ language: python
 matrix:
     include:
         - python: 2.7
-          env: KERAS_BACKEND=tensorflow
+          env:
+            - KERAS_BACKEND=tensorflow
+            - TENSORFLOW_V=1.0.1
```

Comment:
I think this is OK. I had made two mistakes:
- I didn't see that we do test v1.1 already; it just doesn't show up in the diff for this PR.
- I saw that a TensorFlow v1.2 is available, but it is not considered stable yet so we don't need to support it yet